maps/ministation/uplink_item.dm

```diff
@@ -20,7 +20,7 @@
 	..()
 	item = null // Disabled
 
-/datum/uplink_item/stealthy_tools/syndigolashes/New()
+/datum/uplink_item/stealthy_tools/syndigaloshes/New()
```

Comment:
This IS the map counterpart. Maps would't use datums unless they use it like this. `code/datum/uplink_item.dm` is what you're looking for @Intigracy